src/types/annotator.js

```diff
@@ -186,9 +187,42 @@
  */
 
 /**
- * Specifies a content partner/provider to show branding for.
+ * Content provider logo details.
+ *
+ * @typedef ContentInfoLogo
+ * @prop {string} logo
+ * @prop {string} title
+ * @prop {string} link
+ */
+
+/**
+ * Metadata for the current document, for display in the content info banner.
+ *
+ * @typedef ContentInfoItem
+ * @prop {string} title - Title of the current article, chapter etc.
+ * @prop {string} containerTitle - Title of the journal issue, book etc. which
```

Comment:
Would `collectionTitle` be more apropos? Or are you aiming for generic?

Reply:
I was aiming to be bit more generic. Collections/containers I'm aware of include books, journal issues and collections of research reports.